app/views/jobs/_form.html.erb

```diff
@@ -36,6 +36,12 @@
     <p class="pure-form-message">What is the duration for this project?</p>
   </div>
 
+  <div class="uk-margin">
+    <h4><%= form.label :enagagement_type, :class => "uk-form-label" %></h4>
+    <%= form.select :duration, {"Full-time" => "Full-Time", "Part-time" => "Part-Time", "Gig" => "Gig"}, {}, class: "uk-select" %>
```

Comment:
@Robbieakoto Let's change what ends up in the database to use values such as `full_time`, `part_time`, `gig` so they look like enums but aren't.